internal/operands/common-templates/reconcile_test.go

```diff
@@ -30,8 +31,15 @@ var (
 )
 
 const (
-	namespace = "kubevirt"
-	name      = "test-ssp"
+	namespace             = "kubevirt"
+	name                  = "test-ssp"
+	templateVersionLabel  = "template.kubevirt.io/version"
```

Comment:
These labels should be defined in the `common-templates` package code instead of the tests